contracts/helpers/mellow/MellowWithdrawalPhantomToken.sol

```diff
@@ -21,7 +21,7 @@ contract MellowWithdrawalPhantomToken is PhantomERC20, Ownable, IPhantomToken {
 
     bytes32 public constant override contractType = "PHANTOM_TOKEN::MELLOW_WITHDRAWAL";
 
-    uint256 public constant override version = 3_11;
+    uint256 public constant override version = 3_12;
```

Comment:
This is a part of the following PR: https://github.com/Gearbox-protocol/integrations-v3/pulls/174